src/rust/bitbox02-rust/src/keystore.rs

```diff
@@ -898,12 +899,10 @@ mod tests {
             .collect();
 
         for size in [16, 32] {
-            mock_memory();
-            bitbox02::random::fake_reset();
-            bitbox02::memory::set_salt_root(&mock_salt_root).unwrap();
-            lock();
-
             let mut hal = TestingHal::new();
+            let _ = hal.memory.reset_hww();
```

Comment:
Why is this call needed?

If needed, prefer `reset_hww().unwrap()` instead of ignoring the output.